src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application "
+      method="post"
+      data-qa="big">
+      <img class="imgBig"
+      src="./images/Search.svg" alt="magnifying glass">
```

Comment:
I will not correct all formatting mistakes, try to find them yourself by carefully reading the checklist)
```suggestion
      <img 
        class="imgBig"
        src="./images/Search.svg" 
        alt="magnifying glass"
      >
```